vendor/assets/javascripts/leaflet.js.erb

```diff
@@ -0,0 +1,13922 @@
+//= depend_on_asset "marker-icon-2x.png"
```

Comment:
Generally i would prefer an approach that didn't include this in vendor. However, I realize there were some issues that were encountered.